
time="2025-07-27T02:51:49-03:00" level=warning msg="a network with name payment-processor exists but was not created for project \"mxlange\".\nSet `external: true` to use an existing network"
 Container mxlange-db-1  Creating
 Container mxlange-db-1  Created
 Container mxlange-api2-1  Creating
 Container mxlange-api1-1  Creating
 Container mxlange-api2-1  Created
 Container mxlange-api1-1  Created
 Container rinha-nginx  Creating
 Container rinha-nginx  Created
Attaching to api1-1, api2-1, db-1, rinha-nginx
db-1         | The files belonging to this database system will be owned by user "postgres".
db-1         | This user must also own the server process.
db-1         | 
db-1         | The database cluster will be initialized with locale "en_US.utf8".
db-1         | The default database encoding has accordingly been set to "UTF8".
db-1         | The default text search configuration will be set to "english".
db-1         | 
db-1         | Data page checksums are disabled.
db-1         | 
db-1         | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1         | creating subdirectories ... ok
db-1         | selecting dynamic shared memory implementation ... posix
db-1         | selecting default "max_connections" ... 100
db-1         | selecting default "shared_buffers" ... 128MB
db-1         | selecting default time zone ... UTC
db-1         | creating configuration files ... ok
db-1         | running bootstrap script ... ok
db-1         | sh: locale: not found
db-1         | 2025-07-27 05:51:52.420 UTC [35] WARNING:  no usable system locales were found
db-1         | performing post-bootstrap initialization ... ok
db-1         | initdb: warning: enabling "trust" authentication for local connections
db-1         | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1         | syncing data to disk ... ok
db-1         | 
db-1         | 
db-1         | Success. You can now start the database server using:
db-1         | 
db-1         |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1         | 
db-1         | waiting for server to start....2025-07-27 05:51:53.993 UTC [41] LOG:  starting PostgreSQL 17.5 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
db-1         | 2025-07-27 05:51:53.993 UTC [41] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1         | 2025-07-27 05:51:53.995 UTC [44] LOG:  database system was shut down at 2025-07-27 05:51:53 UTC
db-1         | 2025-07-27 05:51:53.999 UTC [41] LOG:  database system is ready to accept connections
db-1         |  done
db-1         | server started
db-1         | 
db-1         | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
db-1         | 
db-1         | waiting for server to shut down....2025-07-27 05:51:54.068 UTC [41] LOG:  received fast shutdown request
db-1         | 2025-07-27 05:51:54.068 UTC [41] LOG:  aborting any active transactions
db-1         | 2025-07-27 05:51:54.081 UTC [41] LOG:  background worker "logical replication launcher" (PID 47) exited with exit code 1
db-1         | 2025-07-27 05:51:54.083 UTC [42] LOG:  shutting down
db-1         | 2025-07-27 05:51:54.083 UTC [42] LOG:  checkpoint starting: shutdown immediate
db-1         | 2025-07-27 05:51:54.084 UTC [42] LOG:  checkpoint complete: wrote 3 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.001 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB; lsn=0/14ED2B8, redo lsn=0/14ED2B8
db-1         | 2025-07-27 05:51:54.091 UTC [41] LOG:  database system is shut down
db-1         |  done
db-1         | server stopped
db-1         | 
db-1         | PostgreSQL init process complete; ready for start up.
db-1         | 
db-1         | 2025-07-27 05:51:54.194 UTC [1] LOG:  starting PostgreSQL 17.5 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
db-1         | 2025-07-27 05:51:54.194 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1         | 2025-07-27 05:51:54.194 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1         | 2025-07-27 05:51:54.194 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1         | 2025-07-27 05:51:54.196 UTC [55] LOG:  database system was shut down at 2025-07-27 05:51:54 UTC
db-1         | 2025-07-27 05:51:54.202 UTC [1] LOG:  database system is ready to accept connections
api1-1       | Connecting to database with connection string: postgresql://postgres:postgres@db:5432/postgres?sslmode=disable
api2-1       | Connecting to database with connection string: postgresql://postgres:postgres@db:5432/postgres?sslmode=disable
db-1         | 2025-07-27 05:51:57.172 UTC [67] ERROR:  duplicate key value violates unique constraint "pg_type_typname_nsp_index"
db-1         | 2025-07-27 05:51:57.172 UTC [67] DETAIL:  Key (typname, typnamespace)=(payments_to_send, 2200) already exists.
db-1         | 2025-07-27 05:51:57.172 UTC [67] STATEMENT:  CREATE TABLE IF NOT EXISTS payments_to_send (
db-1         | 				    id           TEXT PRIMARY KEY,
api2-1       | Running with concurrency: 20
api2-1       | 
api2-1       |  ┌───────────────────────────────────────────────────┐ 
api2-1       |  │                   My Rinha API                    │ 
api2-1       |  │                   Fiber v2.52.8                   │ 
db-1         | 				    amount       DOUBLE PRECISION NOT NULL,
api1-1       | Error creating tables: error creating payments_to_send table: ERROR: duplicate key value violates unique constraint "pg_type_typname_nsp_index" (SQLSTATE 23505)
api2-1       |  │               http://127.0.0.1:8080               │ 
db-1         | 				    requested_at TIMESTAMP WITH TIME ZONE NOT NULL,
db-1         | 				    is_default   BOOLEAN NOT NULL DEFAULT false
db-1         | 				);
api2-1       |  │       (bound on host 0.0.0.0 and port 8080)       │ 
api2-1       |  │                                                   │ 
api2-1       |  │ Handlers ............. 3  Processes ........... 1 │ 
api2-1       |  │ Prefork ....... Disabled  PID ................. 1 │ 
api2-1       |  └───────────────────────────────────────────────────┘ 
api2-1       | 
api1-1       | panic: error creating payments_to_send table: ERROR: duplicate key value violates unique constraint "pg_type_typname_nsp_index" (SQLSTATE 23505)
api1-1       | 
api1-1       | goroutine 1 [running]:
api1-1       | main.main()
api1-1       | 	/app/main.go:61 +0xa76
[Kapi1-1 exited with code 2
rinha-nginx  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
rinha-nginx  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
rinha-nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
rinha-nginx  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
rinha-nginx  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
rinha-nginx  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
rinha-nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
rinha-nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
rinha-nginx  | /docker-entrypoint.sh: Configuration complete; ready for start up
rinha-nginx  | 2025/07/27 05:51:57 [emerg] 1#1: host not found in upstream "api1:8080" in /etc/nginx/nginx.conf:12
rinha-nginx  | nginx: [emerg] host not found in upstream "api1:8080" in /etc/nginx/nginx.conf:12
[Krinha-nginx exited with code 1
[Kapi2-1 exited with code 2
db-1         | 2025-07-27 05:53:04.456 UTC [1] LOG:  received fast shutdown request
db-1         | 2025-07-27 05:53:04.456 UTC [1] LOG:  aborting any active transactions
db-1         | 2025-07-27 05:53:04.458 UTC [1] LOG:  background worker "logical replication launcher" (PID 58) exited with exit code 1
db-1         | 2025-07-27 05:53:04.458 UTC [53] LOG:  shutting down
db-1         | 2025-07-27 05:53:04.458 UTC [53] LOG:  checkpoint starting: shutdown immediate
db-1         | 2025-07-27 05:53:04.460 UTC [53] LOG:  checkpoint complete: wrote 66 buffers (1.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.002 s; sync files=0, longest=0.000 s, average=0.000 s; distance=205 kB, estimate=205 kB; lsn=0/1520940, redo lsn=0/1520940
db-1         | 2025-07-27 05:53:04.465 UTC [1] LOG:  database system is shut down
[Kdb-1 exited with code 0
